### PR TITLE
Version bump

### PR DESCRIPTION
**Pull request by cquiroz**
This PR updates the version of several of our libraries. In particular, scala.js 0.6.13 is going to be important moving forward

Another small update is a change on travis to only build the distribution packages for non-PR builds

The changes reduce time from app 13m to app 8m

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,15 @@ scala:
   - 2.11.8
 jdk:
   - oraclejdk8
+# These directories are cached to S3 at the end of the build
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot/
```
Comment (jdnavarro) [inline .travis.yml]: I did some benchmarks doing precisely that some months ago but didn't notice any difference in build time (I don't remember if it was for OCS or OCS3 though). In any case this is very useful when there are download timeouts 👍 .

File: project/Settings.scala [hunk 2]
```diff
@@ -29,14 +29,14 @@ object Settings {
...
     val uPickle      = "0.4.0"
-    val booPickle    = "1.2.4"
-    val diode        = "1.0.0"
-    val javaTimeJS   = "0.1.0"
+    val booPickle    = "1.2.5"
+    val diode        = "1.1.0"
+    val javaTimeJS   = "0.2.0"
     val javaLogJS    = "0.1.0"
-    val scalaJQuery  = "1.0-RC6"
+    val scalaJQuery  = "1.0"
```
Comment (jdnavarro) [inline project/Settings.scala]: In case I want to version bump myself in the future, how do you pick the versions? Just look at the latest ones and check that they build correctly together?

File: project/Settings.scala [hunk 3]
```diff
@@ -50,7 +50,7 @@ object Settings {
     val unboundId    = "3.1.1"
     val jwt          = "0.8.1"
     val slf4j        = "1.7.21"
-    val knobs        = "3.8.1a"
+    val knobs        = "3.12.27a"
```
Comment (jdnavarro) [inline project/Settings.scala]: Did you try `http4s-0.15.0`? It was released yesterday but I was wondering whether the build would break with the that version.

File: project/SeqexecEngineModules.scala [hunk 4]
```diff
@@ -29,10 +29,6 @@ trait SeqexecEngineModules {
     .settings(
       libraryDependencies ++= Seq(BooPickle.value) ++ TestLibs.value
     )
-    .jsSettings(
-      scalaJSUseRhino := false,
```
Comment (jdnavarro) [inline project/SeqexecEngineModules.scala]: I thought Rhino was dead long time ago 😄, but TIL it's still quite active. Is scalajs using node by default now?

File: project/Settings.scala (hunk 3 above)
Comment (cquiroz) [inline project/Settings.scala]: That is going to be a bigger update, there are only client side updates. http4s update probably deserve its own PR

File: project/Settings.scala (hunk 2 above)
Comment (cquiroz) [inline project/Settings.scala]: Pretty much 😄
